backend/src/main/java/com/shipping/backend/services/ShippingRequestSenderImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.shipping.backend.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ShippingRequestSenderImpl implements  ShippingRequestSender{
+
+    private final static Logger log = LoggerFactory.getLogger(ShippingRequestSenderImpl.class);
+
+    @Value("${config-queue.name}")
+    private String QUEUE_NAME;
```

Comment:
Why do you have this variable if you are not using it on the class o.o?

Reply:
This variable is useless as you mention, so it was use removed from the code.